webcompat/views.py

```diff
@@ -175,7 +175,18 @@ def create_issue():
                 session[param] = request.args.get(param)
         return render_template('new-issue.html', form=bug_form)
     # copy the form so we can add the full UA string to it.
-    form = request.form.copy()
+    if request.form:
+        form = request.form.copy()
+        # To be legit the form needs a couple of parameters
+        # if one essential is missing, it's a bad request
+        must_parameters = set(['url', 'problem_category', 'description',
```

Comment:
@karlcow just so I understand, we're just checking that these formdata keys exist, not that they have a value, right?